QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151102_002940	There's no way that such a huge amount of people can enter Orsha.{nl}That's why we try to gather and accommodate them here as much as possible.
+QUEST_20151102_002941	Lethargic and fear can be seen in people's eyes.{nl)What would you do if you were the village chief?
+QUEST_20151102_002942	{nl}It's the elderly's complaint..{nl}I worry for the lord..
+QUEST_20151102_002943	Those who are usually lazy are still carefree in these time of danger.{nl}People who were suppose to arrive earlier has yet to come.
+QUEST_20151102_002944	The land has been protected by the ancestors of the family.{nl}I will protect the land, no matter the trouble.{nl}Please lend me your power.
+QUEST_20151102_002945	There has been no case of demons being involved during the several years I spent as a chaser.{nl}I believe we have seen all the demons within the area.
+QUEST_20151102_002946	The rumors about you are incredible.{nl}It has spread all around Orsha.
+QUEST_20151102_002947	I will have to revisit my hometown after this task.{nl}I wish to see my parents after experiencing such a horrible event.
+QUEST_20151102_002948	I don't believe there are more important task over at Orsha than here.
+QUEST_20151102_002949	I pray that all the informants are safe.
+QUEST_20151102_002950	To live in such a world, chaser should be more suitable than a hunter.{nl}
+QUEST_20151102_002951	Please find Urbonas instead of worrying about me.
+QUEST_20151102_002952	Please save us from Marnox's grasp.
+QUEST_20151102_002953	I miss the days of the peaceful Orsha.{nl}Those days are never coming back right?
+QUEST_20151102_002954	The demons are coming down from the west.{nl}The events occurred at Ashaq Basement Prison was just the beginning.{nl}How many more sacrifices will have to be done..{nl}I wonder how much longer we can protect the Ashaq.
+QUEST_20151102_002955	All the wounds have yet to heal.{nl}We will leave when we are able to walk.
+QUEST_20151102_002956	It's awful to believe that I could've disappeared.
+QUEST_20151102_002957	By any chance have you seen my cargo left at the Woods of Attached Bridge?{nl}It should've had my mother's keepsake and other various items..
+QUEST_20151102_002958	When you leave this area, you will become a priest that will further devote to the goddess' words..{nl}I pray that you return safely.
+QUEST_20151102_002959	Chaser Daramouse
+QUEST_20151102_002960	What kind of wealth and honor is Thorbana enjoying and followed me here to a such a place..{nl}I must apologize to Thorbana when I safely return from this place.
+QUEST_20151102_002961	Chaser Thorbana
+QUEST_20151102_002962	I followed here since Pranas usually has a acquaintance..{nl}But I did not know the fact that he was someone acting on such a reckless behaviors.
+QUEST_20151102_002963	Before the kingdom's capital has been ruined, kingdom's dispatched officers has been ransacking and annoyed us..{nl}Now those times are missed.
```

Comment:
has been ruined -> was ruined
kingdom's dispatched officers has been -> the kingdom's dispatched officers have been 
annoyed -> annoying

Now those times are missed -> Now those times are gone.(?)